test/routes.spec.js

```diff
@@ -0,0 +1,97 @@
+const chai = require('chai');
+const should = chai.should();
+const chaiHttp = require('chai-http');
+const server = require('../server');
+
+chai.use(chaiHttp);
+
+describe('Client Routes', () => {
+  it('should return a static site from the home route', (done) => {
+    chai.request(server)
+      .get('/')
+      .end((error, response) => {
+        response.should.have.status(200);
+        response.should.be.html;
+        response.res.text.should.be.a('string');
+        done();
+      });
+  });
+
+  it('should return 404 from an undefined route', (done) => {
+    chai.request(server)
+      .get('/ninjas')
+      .end((error, response) => {
+        response.should.have.status(404);
+        done();
+      });
+  });
+
```

Comment:
Solid work on these client test everything looks clear and understandable to me!